automated/linux/gpiod/gpiod.sh

```diff
@@ -0,0 +1,8 @@
+#/bin/bash
```

Comment:
SC1113: Use #!, not just #, for the shebang.

Sanity check passed incorrectly. Because the typo in shebang line, the file was detected as 'plain/text'. I will patch validate.py to treat files end with .sh as shell script.